### ISSUE TITLE
Updating multiple instances

### ISSUE THREAD

**Issue by zackwasli** [FlowFuse/flowfuse#3841]
### Description

Currently you can update the image of Node-Red, but it would helpful to be able to update the code/flow also.

So, if there are multiple instance of Node-Red using the same template code/nodes, you could update them all.

Requested by:

https://app-eu1.hubspot.com/contacts/26586079/record/0-2/10189549510

### Which customers would this be available to

Team + Enterprise Tiers (EE)

### Have you provided an initial effort estimate for this issue?

I can not provide an initial effort estimate

**Comment by knolleary** [comment]
My initial interpretation of this request suggests a suitable response would be to add support for a Pipeline to deploy to multiple instances in a single stage.

But that may be misunderstanding of the request. I think we need to dig into this a bit more to understand the scenario as it can be interpreted in a few different ways - and thus satisfied in different ways. We would want to make sure we're addressing the true customer need with anything we did in response.

**Comment by zackwasli** [comment]
Agreed, working to get more info on the specifics, asking these questions and will post answers: https://flowforgeworkspace.slack.com/archives/C03FAUBPPJ7/p1715202997759439?thread_ts=1715004346.071099&cid=C03FAUBPPJ7

**Comment by joepavitt** [comment]
This could read similar to https://github.com/FlowFuse/flowfuse/issues/3230 where we have a centralised "store" of some Node-RED code, and that needs to be updated automatically to many instances using that code/subflow. #3841 is larger, as it could also encompass node-red versions, etc.

The other way of interpreting this is that we are being asked for "groups" of instances, similar to how we have groups of devices, and the request is simply that they want to push a single flow out to multiple FF-Hosted instances in one click.

**Comment by zackwasli** [comment]
I would guess the latter, but I asked them these questions: 

https://flowforgeworkspace.slack.com/archives/C03FAUBPPJ7/p1715202997759439?thread_ts=1715004346.071099&cid=C03FAUBPPJ7

on May 8th and have not gotten a reply yet. Will follow up today.
